src/index.js

```diff
@@ -21,26 +21,32 @@ const toggleElement = (element, show) => {
 
 const NOOP = () => {}
 
+const DEFAULT_POSITION = 10
```

Comment:
@rafaeleyng  I'm not sure about this variable name

Reply:
Looks good to me. Maybe `DEFAULT_POSITION_VALUE`, but doesn't make much difference